src/sentry/tasks/post_process.py

```diff
@@ -669,36 +669,49 @@ def process_commits(job: PostProcessJob) -> None:
                 cache.set(has_commit_key, org_has_commit, 3600)
 
             if org_has_commit:
-                group_cache_key = f"w-o-i:g-{event.group_id}"
```

Comment:
What's happening in this block of changes?

I see you're moving the blocking starting with `from sentry.utils.committers import get_frame_paths` up here from the else down there.